libs/designsystem/progress-circle/src/progress-circle-ring.component.scss

```
@@ -1,5 +1,9 @@
 @use '@kirbydesign/core/src/scss/utils';
 
+svg {
+  transform: rotate(-90deg);
```

Totally optional suggestion:

```suggestion
  rotate: -90deg;
```
https://developer.mozilla.org/en-US/docs/Web/CSS/rotate